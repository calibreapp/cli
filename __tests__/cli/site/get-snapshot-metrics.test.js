import {
  runCLI,
  setupIntegrationServer,
  teardownIntegrationServer
} from '../../utils'

import snapshotMetrics from '../../fixtures/snapshotMetrics-no-snapshot.json'

describe('snapshot metrics', () => {
  beforeAll(() => setupIntegrationServer(snapshotMetrics))
  afterAll(() => teardownIntegrationServer())

  test('returns no snapshot error', async () => {
    const out = await runCLI({
      args: 'site get-snapshot-metrics --site=test --snapshot=1000'
    })
    expect(out).toMatchSnapshot()
  })
})
