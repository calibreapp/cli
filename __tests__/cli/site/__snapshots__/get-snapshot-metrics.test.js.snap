// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot metrics returns no snapshot error 1`] = `
"- Connecting to Calibre
✖ Connecting to Calibre


 Error: Snapshot not found 

 calibre site get-snapshot-metrics [options]

Get the metrics of a given snapshot

Options:

  --help      Show help                                                [boolean]
  --version   Show version number                                      [boolean]
  --site      A unique slug identifying each Site. You can find it in Site Setti
              ngs → General or by using the calibre site list command.
                                                             [string] [required]
  --snapshot  The identifying id of a snapshot                        [required]
  --json      Outputs the results of the command in JSON format.
  --csv       Outputs the results of the command in CSV format."
`;
